starter-kit/app/src/components/Home/HomeView.js

```diff
@@ -4,13 +4,14 @@ import "./Home.css";
 
 const HomeView = ({ bodyRef, columns, data }) => {
   return (
-    <div className="container">
+    <div className="container"  style={{ display: "grid", gridTemplateColumns: "repeat(2, 1fr)"}} >
```

Comment:
Inline styling? Also please note that the Home page is going to change once the layout task is delivered. So hopefully any styling to the home page is not linked to the pagination work.